src/components/Article/Article.tsx

```diff
@@ -130,16 +130,18 @@ const Article = ({
   // the article doesn't exist on initial page load (At least without SSR).
   useEffect(() => {
     if (location.hash && article.content) {
-      const element = document.getElementById(location.hash.slice(1));
-      const elementTop = element?.getBoundingClientRect().top ?? 0;
-      const bodyTop = document.body.getBoundingClientRect().top ?? 0;
-      const absoluteTop = elementTop - bodyTop;
-      const scrollPosition = absoluteTop - MastheadHeightPx;
+      setTimeout(() => {
+        const element = document.getElementById(location.hash.slice(1));
+        const elementTop = element?.getBoundingClientRect().top ?? 0;
+        const bodyTop = document.body.getBoundingClientRect().top ?? 0;
+        const absoluteTop = elementTop - bodyTop;
+        const scrollPosition = absoluteTop - MastheadHeightPx - 16;
```

Comment:
Lokalt hos meg gir dette et bra resultat, men sjekk gjerne sjølv.
```suggestion
        const scrollPosition = absoluteTop - MastheadHeightPx - 90;
```

Reply:
Satte den til 2x høyden på Masthead. Det er ca det samme som her